src/Microsoft.Health.SqlServer/Features/Client/SqlConnectionWrapper.cs

```diff
@@ -51,15 +52,15 @@ public SqlTransaction SqlTransaction
             }
         }
 
-        internal async Task InitializeAsync()
+        internal async Task InitializeAsync(CancellationToken cancellationToken)
         {
             if (_enlistInTransactionIfPresent && _sqlTransactionHandler.SqlTransactionScope?.SqlConnection != null)
             {
                 _sqlConnection = _sqlTransactionHandler.SqlTransactionScope.SqlConnection;
             }
             else
             {
-                _sqlConnection = await _sqlConnectionFactory.GetSqlConnectionAsync();
+                _sqlConnection = await _sqlConnectionFactory.GetSqlConnectionAsync(cancellationToken: cancellationToken);
```

Comment:
nit: don't need named parameter here

Reply:
Actually I just tried it. Since the first argument is InitialCatalog, it considers the ConnectionToken as InitialCatalog if named parameter is not used.